byzcoin/service.go

```diff
@@ -604,6 +604,38 @@ func (s *Service) GetSignerCounters(req *GetSignerCounters) (*GetSignerCountersR
 	return &resp, nil
 }
 
+// GetUpdates returns instances that have a newer versions than the ones
+// passed to it.
+func (s *Service) GetUpdates(pr *GetUpdatesRequest) (*GetUpdatesReply, error) {
+	reply := &GetUpdatesReply{}
```

Comment:
I like to delay initialisation until it's really needed (such as before the `for`), I find it more readable as "in the next block, we initialize theses variables".